pages/common/evil-winrm.md

```diff
@@ -0,0 +1,37 @@
+# evil-winrm
+
+> Windows Remote Management (WinRM) shell for pentesting.
+> Once connected, we get a PowerShell prompt on the target host.
+> More information: <https://github.com/Hackplayers/evil-winrm>.
+
+- Connect to a host:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}}`
+
+- Connect to a host, passing the password hash:
+
+`evil-winrm --ip {{ip}} --user {{user}} --hash {{nt_hash}}`
+
+- Connect to a host, specifying folders for scripts and executables:
```

Comment:
specifying _directories_

Reply:
Ah, I didn't even notice that.

And good point - @dhondta, perhaps you could open another PR please?